tests/acceptance/Search.feature

```diff
@@ -168,13 +165,13 @@ Feature: main search function
     And I wait "1" seconds
     Then I should see a link "Data and software to accompany the paper: Applying compressed sensing to genome-wide association studies." to "/dataset/100094"
 
-  @ok
-  Scenario: Search for a term that is only in dataset types
-    Given I am on "/"
-    And I fill in the field of "id" "keyword" with "epigenomic"
-    And I press the button "Search"
-    And I wait "1" seconds
-    Then I should see a link "Genome data from foxtail millet (<em>Setaria italica</em>)." to "/dataset/100020"
+#  @ok
+#  Scenario: Search for a term that is only in dataset types
+#    Given I am on "/"
+#    And I fill in the field of "id" "keyword" with "epigenomic"
+#    And I press the button "Search"
+#    And I wait "1" seconds
+#    Then I should see a link "Genome data from foxtail millet (<em>Setaria italica</em>)." to "/dataset/100020"
```

Comment:
Deleting this test means we are reducing test coverage?

Reply:
Since new dataset `10039` is included, this commented scenario has been reverted.